tests/fixtures/sample_lib/data_model/sample.py

```diff
@@ -29,6 +30,18 @@ class SampleListInputType(DataObjectBase):
     inputs: List[SampleInputType]
 
 
+# Test w/ just import and no dataobject
+@dataobject(package="caikit_data_model.sample_lib")
+class JsonDictInputType(DataObjectBase):
```

Comment:
where / how is this used?

Reply:
@joerunde see below.  I'd probably remove it, but I think @prashantgupta24 wanted it kept in.  It does impact the descriptor pool which affects proto3 as described below.

Commented best I could, but "less is more"?  Just asking @prashantgupta24 if I should remove it.  He might want to add.